arch/TMS320C64x/TMS320C64xMapping.c

```diff
@@ -1720,152 +1720,152 @@ void TMS320C64x_get_insn_id(cs_struct *h, cs_insn *insn, unsigned int id)
+	{TMS320C64X_INS_MPYHL, "mpyhl"},
+	{TMS320C64X_INS_MPYHLU, "mpyhlu"},
+	{TMS320C64X_INS_MPYHSLU, "mpyhslu"},
+	{TMS320C64X_INS_MPYHSU, "mpyhsu"},
+	{TMS320C64X_INS_MPYHU, "mpyhu"},
+	{TMS320C64X_INS_MPYHULS, "mpyhuls"},
+	{TMS320C64X_INS_MPYHUS, "mpyhus"},
+	{TMS320C64X_INS_MPYLH, "mpylh"},
+	{TMS320C64X_INS_MPYLHU, "mpylhu"},
+	{TMS320C64X_INS_MPYLI, "mpyli"},
+	{TMS320C64X_INS_MPYLIR, "mpylir"},
+	{TMS320C64X_INS_MPYLSHU, "mpylshu"},
+	{TMS320C64X_INS_MPYLUHS, "mpyluhs"},
+	{TMS320C64X_INS_MPYSU, "mpysu"},
+	{TMS320C64X_INS_MPYSU4, "mpysu4"},
+	{TMS320C64X_INS_MPYU, "mpyu"},
+	{TMS320C64X_INS_MPYU4, "mpyu4"},
+	{TMS320C64X_INS_MPYUS, "mpyus"},
+	{TMS320C64X_INS_MVC, "mvc"},
+	{TMS320C64X_INS_MVD, "mvd"},
+	{TMS320C64X_INS_MVK, "mvk"},
+	{TMS320C64X_INS_MVKL, "mvkl"},
+	{TMS320C64X_INS_MVKLH, "mvklh"},
+	{TMS320C64X_INS_NOP, "nop"},
+	{TMS320C64X_INS_NORM, "norm"},
+	{TMS320C64X_INS_OR, "or"},
+	{TMS320C64X_INS_PACK2, "pack2"},
+	{TMS320C64X_INS_PACKH2, "packh2"},
+	{TMS320C64X_INS_PACKH4, "packh4"},
+	{TMS320C64X_INS_PACKHL2, "packhl2"},
+	{TMS320C64X_INS_PACKL4, "packl4"},
+	{TMS320C64X_INS_PACKLH2, "packlh2"},
+	{TMS320C64X_INS_ROTL, "rotl"},
+	{TMS320C64X_INS_SADD, "sadd"},
+	{TMS320C64X_INS_SADD2, "sadd2"},
+	{TMS320C64X_INS_SADDU4, "saddu4"},
+	{TMS320C64X_INS_SADDUS2, "saddus2"},
+	{TMS320C64X_INS_SAT, "sat"},
+	{TMS320C64X_INS_SET, "set"},
+	{TMS320C64X_INS_SHFL, "shfl"},
+	{TMS320C64X_INS_SHL, "shl"},
+	{TMS320C64X_INS_SHLMB, "shlmb"},
+	{TMS320C64X_INS_SHR, "shr"},
+	{TMS320C64X_INS_SHR2, "shr2"},
+	{TMS320C64X_INS_SHRMB, "shrmb"},
+	{TMS320C64X_INS_SHRU, "shru"},
+	{TMS320C64X_INS_SHRU2, "shru2"},
+	{TMS320C64X_INS_SMPY, "smpy"},
+	{TMS320C64X_INS_SMPY2, "smpy2"},
+	{TMS320C64X_INS_SMPYH, "smpyh"},
+	{TMS320C64X_INS_SMPYHL, "smpyhl"},
+	{TMS320C64X_INS_SMPYLH, "smpylh"},
+	{TMS320C64X_INS_SPACK2, "spack2"},
+	{TMS320C64X_INS_SPACKU4, "spacku4"},
+	{TMS320C64X_INS_SSHL, "sshl"},
+	{TMS320C64X_INS_SSHVL, "sshvl"},
+	{TMS320C64X_INS_SSHVR, "sshvr"},
+	{TMS320C64X_INS_SSUB, "ssub"},
+	{TMS320C64X_INS_STB, "stb"},
+	{TMS320C64X_INS_STDW, "stdw"},
+	{TMS320C64X_INS_STH, "sth"},
+	{TMS320C64X_INS_STNDW, "stndw"},
+	{TMS320C64X_INS_STNW, "stnw"},
+	{TMS320C64X_INS_STW, "stw"},
+	{TMS320C64X_INS_SUB, "sub"},
+	{TMS320C64X_INS_SUB2, "sub2"},
+	{TMS320C64X_INS_SUB4, "sub4"},
+	{TMS320C64X_INS_SUBAB, "subab"},
+	{TMS320C64X_INS_SUBABS4, "subabs4"},
+	{TMS320C64X_INS_SUBAH, "subah"},
+	{TMS320C64X_INS_SUBAW, "subaw"},
+	{TMS320C64X_INS_SUBC, "subc"},
+	{TMS320C64X_INS_SUBU, "subu"},
+	{TMS320C64X_INS_SWAP4, "swap4"},
+	{TMS320C64X_INS_UNPKHU4, "unpkhu4"},
+	{TMS320C64X_INS_UNPKLU4, "unpklu4"},
+	{TMS320C64X_INS_XOR, "xor"},
+	{TMS320C64X_INS_XPND2, "xpnd2"},
+	{TMS320C64X_INS_XPND4, "xpnd4"},
+	{TMS320C64X_INS_IDLE, "idle"},
+	{TMS320C64X_INS_MV, "mv"},
+	{TMS320C64X_INS_NEG, "neg"},
+	{TMS320C64X_INS_NOT, "not"},
+	{TMS320C64X_INS_SWAP2, "swap2"},
+	{TMS320C64X_INS_ZERO, "zero"},
+	{TMS320C64X_INS_ENDING, "ending"},
```

Comment:
this ENDING is just to mark end of list of instructions, but not a real instruction, so it should not be here.
please remove it

Reply:
indeed